File: PaketCay.java
package tr.k12.meb.ibrahimonalfenlisesi;

import java.util.Scanner;

public class PaketCay {

    public static void main(String[] args) {
        int paketCay=10;
        int paketSayisi;
        float indirim;
        System.out.print("Paket sayısı giriniz: ");
        Scanner oku=new Scanner(System.in);
        paketSayisi=oku.nextInt();
        if ((paketSayisi>=10)&&(paketSayisi<50)){
            indirim=(float)0.05;
        }else  if ((paketSayisi>=50)&&(paketSayisi<100)){
            indirim=(float)0.10;
        }else {
            indirim=(float)0.15;
        }
        double toplam=paketCay*paketSayisi;
        double kdv=toplam*0.08;
        double satis=(toplam+kdv)-(toplam+kdv)*indirim;
        System.out.printf("Toplam:\t%+10.2f TL%nKDV: \t%+10.2f TL%nİndirim:%10.2f TL%nSatiş:\t%+10.2f TL",toplam,kdv,-1*(toplam+kdv)*indirim,satis);
    }

}

File: paketcay.py
#çözüm 1
paketCay=10
paketSayisi=int(input("Paket sayısını giriniz :"))
if paketSayisi>=10 and paketSayisi<50:
    indirim=0.05
elif paketSayisi>=50 and paketSayisi<100:
    indirim=0.10
else:
    indirim=0.15
toplam=paketCay*paketSayisi
kdv=toplam*0.08
satis=(toplam+kdv)-(toplam+kdv)*indirim
print("Toplam\t: {0}TL\nKDV\t: {1}TL\nİndirim\t:{2}TL\nSatış\t: {3}TL".format(toplam,kdv,-(toplam+kdv)*indirim,satis))
#çözüm 2
paketCay=10
paketSayisi=int(input("Paket sayısını giriniz :"))
if paketSayisi>=10 and paketSayisi<50:
    indirim=0.95
elif paketSayisi>=50 and paketSayisi<100:
    indirim=0.90
else:
    indirim=0.85
toplam=paketCay*paketSayisi
kdv=toplam*0.08
satis=(toplam+kdv)*indirim
print("Toplam\t: {0}TL\nKDV\t: {1}TL\nİndirim\t:-{2}TL\nSatış\t: {3}TL".format(toplam,kdv,(toplam+kdv)-satis,satis))

